### ISSUE TITLE
self.cards seems too long

### ISSUE THREAD

**Issue by giladgressel** [popo26/Projects#1]
https://github.com/popo26/Projects/blob/e47a785bd2c1fb9647c97a3738f2c8824f7d09e5/301_03_build_a_card_game/cards.py#L13-L18

So you have this super long list and it repeats itself. I'm not clear you needed to make it so long and repeating?
it couldn't be done with just 1 round?

regardless even if you _do_ need all of them (4x), then use multi-lines. You need to break your habit of coding off the side of the screen.

**Comment by popo26** [comment]
True! Yes I missed this one., Thanks for pointing out. i wanted to have 51 cards so I added all.
